pipelines/sqre/verify_drp_metrics.groovy

```diff
@@ -39,7 +34,7 @@ notify.wrap {
 
   // run multiple datasets, if defined, in parallel
   def jobConf     = drp
-  def jobConfName = 'validate_drp_gen3'
+  def jobConfName = 'verify_drp_metrics'
```

Comment:
If you actually want this to run again (are the NaN issues fixed?), you'll need to get rid of the disablement in (new) lines 66/67.

Reply:
I'm hoping the NaN issues will be fixed in this weekly.